packages/website-frontend/package.json

```diff
@@ -16,36 +13,20 @@
   "main": "main.js",
   "typings": "main.d.ts",
   "devDependencies": {
-    "@angular-devkit/build-angular": "18.2.1",
-    "@angular/animations": "18.2.0",
-    "@angular/cli": "18.2.1",
-    "@angular/common": "18.2.0",
-    "@angular/compiler": "18.2.0",
-    "@angular/compiler-cli": "18.2.0",
-    "@angular/core": "18.2.0",
-    "@angular/forms": "18.2.0",
-    "@angular/platform-browser": "18.2.0",
-    "@angular/platform-browser-dynamic": "18.2.0",
-    "@angular/router": "18.2.0",
-    "@github/clipboard-copy-element": "1.3.0",
-    "@primer/octicons": "19.11.0",
     "@stryker-mutator/dashboard-common": "0.14.3",
     "@stryker-mutator/dashboard-contract": "0.14.3",
-    "@stryker-mutator/karma-runner": "8.5.0",
+    "@stryker-mutator/karma-runner": "8.2.6",
```

Comment:
I think this can be removed? I think you also accidentally reverted some dependencies to their older versions

Reply:
Yep, rebasing borked the package.json, so I picked an older version - will fix.